content/pt-br/cloud_native_tech.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Tecnologia Nativa da Nuvem
+status: Completed
+category: Conceito
+---
+
+## O que é
+
+As tecnologias nativas da nuvem, também conhecidas como a pilha nativa da nuvem, são as tecnologias usadas para criar [aplicativos nativos da nuvem](/cloud_native_apps/). Essas tecnologias permitem que as organizações criem e executem aplicações escaláveis em ambientes modernos e dinâmicos, tais como as nuvens públicas, privadas e  híbridas, ao mesmo tempo em que aproveitam ao máximo os benefícios da [computação em nuvem](/cloud_computing/). Eles são projetados desde o início para explorar os recursos de computação em nuvem e contêineres, serviços em malha, microsserviços e infraestrutura imutável exemplificam essa abordagem.
```

Comment:
[The `pt-br` version of the term `Cloud Native Apps`](https://deploy-preview-683--cncfglossary.netlify.app/pt-br/cloud_native_apps/) exists, so updated the link. 😊

```suggestion
As tecnologias nativas da nuvem, também conhecidas como a pilha nativa da nuvem, são as tecnologias usadas para criar [aplicativos nativos da nuvem](/pt-br/cloud_native_apps/). Essas tecnologias permitem que as organizações criem e executem aplicações escaláveis em ambientes modernos e dinâmicos, tais como as nuvens públicas, privadas e  híbridas, ao mesmo tempo em que aproveitam ao máximo os benefícios da [computação em nuvem](/cloud_computing/). Eles são projetados desde o início para explorar os recursos de computação em nuvem e contêineres, serviços em malha, microsserviços e infraestrutura imutável exemplificam essa abordagem.
```

Reply:
Sorry about that. I will keep an eye out.